unicodetools/data/ucd/dev/extracted/DerivedName.txt

```diff
@@ -31364,6 +31364,26 @@ FFFD          ; REPLACEMENT CHARACTER
 116C7         ; TAKRI DIGIT SEVEN
 116C8         ; TAKRI DIGIT EIGHT
 116C9         ; TAKRI DIGIT NINE
+116D0         ; MYANMAR PAO DIGIT ZERO
+116D1         ; MYANMAR PAO DIGIT ONE
+116D2         ; MYANMAR PAO DIGIT TWO
+116D3         ; MYANMAR PAO DIGIT THREE
+116D4         ; MYANMAR PAO DIGIT FOUR
+116D5         ; MYANMAR PAO DIGIT FIVE
+116D6         ; MYANMAR PAO DIGIT SIX
+116D7         ; MYANMAR PAO DIGIT SEVEN
+116D8         ; MYANMAR PAO DIGIT EIGHT
+116D9         ; MYANMAR PAO DIGIT NINE
+116DA         ; MYANMAR EASTERN PWO KAREN DIGIT ZERO
```

Comment:
FYI @Ken-Whistler -- aside from the Math digits (e.g., U+1D7CE), this is the first set of decimal digits where the zero digit does not have a ...0 or ...6 code point. Do we care?

Reply:
We have talked about this a few times (at SAH and UTC) and we are fine with it. UK objected to this at JTC1/SC2 but their comment was rejected.